src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.tsx

```diff
@@ -61,24 +65,14 @@ export const DefaultTranscriptListItem = (
   const transcriptStartX = scale(relativeTranscriptStart) as number;
   const transcriptWidth = scale(transcriptLength) as number;
 
-  const defaultTranscriptLabelMap = {
-    selected: {
-      label: 'Selected',
-      helpText:
-        'The selected transcript is a default single transcript per protein coding gene that is representative of biology, well-supported, expressed and highly conserved'
-    }
-  };
-
-  const canonicalType = 'selected'; // TODO Change this to transcript.mane/plus etc when available
-
   return (
     <div className={styles.defaultTranscriptListItem}>
       <div className={transcriptsListStyles.row}>
         {props.isDefault && (
           <div className={styles.defaultTranscriptLabel}>
-            <span>{defaultTranscriptLabelMap[canonicalType]?.label}</span>
+            <span>{DefaultTranscriptLabelMap[CanonicalType]?.label}</span>
             <QuestionButton
-              helpText={defaultTranscriptLabelMap[canonicalType]?.helpText}
+              helpText={DefaultTranscriptLabelMap[CanonicalType]?.helpText}
             />
```

Comment:
I am confused. The `DefaultTranscript.tsx` file is not a React component (as it currently stands, it doesn't even need to be a tsx file; it could be a ts if there's no React code there). Wasn't the React code above also supposed to go to that component?

Also, it would be probably misleading to call that component `DefaultTranscript`. What is it that this label is about? Is it `TranscriptConfidenceLabel` or something?

Reply:
yes there is no react code so I should be using .ts. BTW I didnt understand your comment on moving the above react code.

Naming - atm the config contains label and helptext for different transcript types. It shows the confidence/quality of that transcript, so what would be the best name? open to suggestions

Reply:
> I didnt understand your comment on moving the above react code.

I thought this bit of code:

```
          <div className={styles.defaultTranscriptLabel}>
            <span>{DefaultTranscriptLabelMap[CanonicalType]?.label}</span>
            <QuestionButton
              helpText={DefaultTranscriptLabelMap[CanonicalType]?.helpText}
            />
          </div>
```

was going to be extracted as a component. It would receive transcript as a prop, and would have the auxiliary code to decide which label to show for such transcript and which text to pass to the QuestionButton.

Reply:
Ok got it.